ideascube/search/templates/search/filter_by_package_id.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n ideascube_tags %}
+{% if available_kinds %}
+<div class="card tinted">
+    <h4><span class="theme discover">{% trans "browse" %}</span> {% trans 'by package' %}</h4>
+    {% for id, label in available_package_ids.items %}
```

Comment:
I think we are fine for this time, but we'll quickly face the issue of having too much package installed.
Also, I'm not sure the concept of "package" is really end user ready. I'll think about a suggestion.